lambda/s3_flush_lambda.py

```diff
@@ -134,7 +134,11 @@
         existing_cube.from_blosc(existing_cube_bytes, new_cube.time_range)
 
         # Merge cuboids
-        existing_cube.overwrite(new_cube.data, new_cube.time_range)
+        if new_cube.is_bool():
```

Comment:
Is this really a good test? Could an annotation ever look like a bool cube?

Reply:
Yes an annotation could look like a bool cube, and I share your concern with how I've implemented the check. In the edge case where a user uploads a cutout consisting of solely 0s and 1s, the check would pass and it would overwrite to black instead of the expected behavior of posting the cutout. 

I'm currently trying to figure out another way to check if the cube originated from a `cutout_to_black` call or from a `post_cutout` call. Will update with progress.

Reply:
I think the way to go is to put a flag in the SQS message that indicates that the data identified by the `write_cuboid_key` is for a black cutout.